features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/ajax/loganalyzer.jag

```diff
@@ -0,0 +1,13 @@
+<%
+include("/jagg/jagg.jag");
```

Comment:
Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD